docs/source/dynamic_typing.rst

```diff
@@ -86,7 +86,7 @@ treated as ``Any``:
         reveal_type(x[0])     # Revealed type is "Any"
         x[0].anything_goes()  # OK
 
-You can make mypy warn you about untyped function parameters using the
+You can make mypy warn you about unsaturated generic parameters using the
```

Comment:
```suggestion
You can make mypy warn you about missing generic parameters using the
```
Don't think we use the term "unsaturated"

Reply:
Sure thing, thanks!